src/reactapp/public/i18n/en_US.json

```diff
@@ -60,5 +60,6 @@
   "Street":"Street",
   "Total":"Total",
   "Update":"Update",
-  "{} zipcode: {}":"{} zipcode: {}"
+  "{} zipcode: {}":"{} zipcode: {}",
```

Comment:
Out of curiosity: why did you decide to use `{}` instead of the Magento convention `%s` ?

Reply:
No reason other than `{}` feel like a placeholder for a javascript developer. So for example

```
firstname = 'rajeev';
lastname = 'tomy';
console.log(`my firstname is ${firstname} and last name is ${lastname}`) 
```
So `${}` is a kind of placeholder. `{}` is close to that. So i thought to use it.

It is easy for me to change it to `%s`. Do you need it?

Reply:
No, was only curious. I can imagine reusing magento translations is easier if you kept the same strings from core :) What do you think @schmengler?

Reply:
True but it would need some processing anyways because we use JSON files here, so I'm fine with it like this. `%s` is not very JavaScript'y